# Fix FTBFS with gcc 4.4.
# Modified from a patch by Martin Michlmayr
--- a/src/display/utils.h
+++ b/src/display/utils.h
@@ -37,6 +37,7 @@
 #ifndef RTORRENT_DISPLAY_UTILS_H
 #define RTORRENT_DISPLAY_UTILS_H
 
+#include <cstdio>
 #include <ctime>
 #include <string>
 
--- a/src/rpc/parse.cc
+++ b/src/rpc/parse.cc
@@ -38,6 +38,7 @@
 
 #include <cstring>
 #include <locale>
+#include <cstdio>
 #include <rak/path.h>
 #include <torrent/exceptions.h>
 
--- a/src/rpc/scgi_task.cc
+++ b/src/rpc/scgi_task.cc
@@ -36,6 +36,7 @@
 
 #include "config.h"
 
+#include <cstdio>
 #include <rak/error_number.h>
 #include <sys/types.h>
 #include <sys/socket.h>
--- a/src/utils/lockfile.cc
+++ b/src/utils/lockfile.cc
@@ -39,6 +39,7 @@
 #include <algorithm>
 #include <cctype>
 #include <cerrno>
+#include <cstdio>
 #include <cstring>
 #include <sstream>
 #include <fcntl.h>
--- a/src/command_network.cc
+++ b/src/command_network.cc
@@ -36,6 +36,7 @@
 
 #include "config.h"
 
+#include <cstdio>
 #include <functional>
 #include <rak/address_info.h>
 #include <rak/path.h>
--- a/src/command_ui.cc
+++ b/src/command_ui.cc
@@ -36,6 +36,7 @@
 
 #include "config.h"
 
+#include <cstdio>
 #include <ctime>
 #include <rak/functional.h>
 #include <rak/functional_fun.h>
--- a/src/command_download.cc
+++ b/src/command_download.cc
@@ -36,6 +36,7 @@
 
 #include "config.h"
 
+#include <cstdio>
 #include <functional>
 #include <unistd.h>
 #include <rak/file_stat.h>
--- a/src/command_dynamic.cc
+++ b/src/command_dynamic.cc
@@ -36,6 +36,7 @@
 
 #include "config.h"
 
+#include <cstdio>
 #include <algorithm>
 
 #include "globals.h"
--- a/src/command_events.cc
+++ b/src/command_events.cc
@@ -36,6 +36,7 @@
 
 #include "config.h"
 
+#include <cstdio>
 #include <functional>
 #include <rak/file_stat.h>
 #include <rak/path.h>
--- a/src/command_file.cc
+++ b/src/command_file.cc
@@ -36,6 +36,7 @@
 
 #include "config.h"
 
+#include <cstdio>
 #include <rak/error_number.h>
 #include <rak/path.h>
 #include <torrent/data/file.h>
--- a/src/command_helpers.cc
+++ b/src/command_helpers.cc
@@ -36,6 +36,7 @@
 
 #include "config.h"
 
+#include <cstdio>
 #include <torrent/exceptions.h>
 
 #include "rpc/command_slot.h"
--- a/src/command_local.cc
+++ b/src/command_local.cc
@@ -36,6 +36,7 @@
 
 #include "config.h"
 
+#include <cstdio>
 #include <fcntl.h>
 #include <functional>
 #include <unistd.h>
--- a/src/command_object.cc
+++ b/src/command_object.cc
@@ -36,6 +36,7 @@
 
 #include "config.h"
 
+#include <cstdio>
 #include <algorithm>
 
 #include "globals.h"
--- a/src/command_peer.cc
+++ b/src/command_peer.cc
@@ -36,6 +36,7 @@
 
 #include "config.h"
 
+#include <cstdio>
 #include <rak/error_number.h>
 #include <rak/path.h>
 #include <rak/socket_address.h>
--- a/src/command_scheduler.cc
+++ b/src/command_scheduler.cc
@@ -36,6 +36,7 @@
 
 #include "config.h"
 
+#include <cstdio>
 #include "core/manager.h"
 #include "core/download_list.h"
 #include "rpc/command_variable.h"
--- a/src/command_tracker.cc
+++ b/src/command_tracker.cc
@@ -36,6 +36,7 @@
 
 #include "config.h"
 
+#include <cstdio>
 #include <rak/error_number.h>
 #include <torrent/tracker.h>
 
